buildSrc/src/main/kotlin/TraceDebuggerTasks.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+@file:Suppress("DEPRECATION", "UNUSED_VARIABLE")
+
+import org.gradle.api.Project
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.api.file.DuplicatesStrategy
+import org.gradle.kotlin.dsl.getByType
+import org.gradle.kotlin.dsl.register
+import org.gradle.api.plugins.JavaPluginExtension
+
+// Below are tasks that are used by the trace debugger plugin.
+// When these jars are loaded the `-Dlincheck.traceDebuggerMode=true` VM argument is expected
```

Comment:
Please add the corresponding check

Reply:
The check is added to the agent premain method. I need to check it in runtime, so I cannot put it right here